specification/2.0/README.adoc

```diff
@@ -0,0 +1,2430 @@
+
+[source,c]
+----
+uint32 chunkLength
+uint32 chunkType
+ubyte[] chunkData
+----
+
+* `chunkLength` is the length of `chunkData`, in bytes.
+* `chunkType` indicates the type of chunk. See <<table-chunktypes>> for details.
+* `chunkData` is a binary payload of chunk.
+
+The start and the end of each chunk must be aligned to 4-byte boundary. See chunks definitions for padding schemes. Chunks must appear in exactly the order given in <<table-chunktypes>>.
+
+[[table-chunktypes]]
+.Chunk types
+[options="header"]
+|====
+|    | Chunk Type | ASCII | Description             | Occurrences
+| 1. | 0x4E4F534A | JSON  | Structured JSON content | 1
+| 2. | 0x004E4942 | BIN   | Binary buffer           | 0 or 1
+|====
+
+Client implementations must ignore chunks with unknown types to enable glTF extensions to reference additional chunks with new types following the first two chunks.
+
+
+[[structured-json-content]]
+==== Structured JSON Content
+
+This chunk holds the structured glTF content description, as it would be provided within a .gltf file.
+
+[NOTE]
+.Implementation Note
+====
+In a JavaScript implementation, the `TextDecoder` API can be used to extract the glTF content from the ArrayBuffer, and then the JSON can be parsed with `JSON.parse` as usual.
+====
+
+This chunk must be the very first chunk of Binary glTF asset. By reading this chunk first, an implementation is able to progressively retrieve resources from subsequent chunks. This way, it is also possible to read only a selected subset of resources from a Binary glTF asset (for instance, the coarsest LOD of a mesh).
+
+This chunk must be padded with trailing `Space` chars (`0x20`) to satisfy alignment requirements.
+
+
+[[binary-buffer]]
+==== Binary buffer
+
+This chunk contains the binary payload for geometry, animation key frames, skins, and images. See glTF specification for details on referencing this chunk from JSON.
+
+This chunk must be the second chunk of the Binary glTF asset.
+
+This chunk must be padded with trailing zeros (`0x00`) to satisfy alignment requirements.
+
+
+[[properties-reference]]
+= Properties Reference
+
+// Generated with wetzel
+include::PropertiesReference.adoc[]
+
+
+[[acknowledgments]]
+= Acknowledgments
+
+* Sarah Chow, Cesium
+* Tom Fili, Cesium
+* Darryl Gough
+* Eric Haines, Autodesk
+* Yu Chen Hou
+* Scott Hunter, Analytical Graphics, Inc.
+* Brandon Jones, Google
+* Sean Lilley, Cesium
+* Juan Linietsky, Godot Engine
+* Matthew McMullan
+* Mohamad Moneimne, University of Pennsylvania
+* Kai Ninomiya, formerly Cesium
+* Cedric Pinson, Sketchfab
+* Jeff Russell, Marmoset
+* Miguel Sousa, Fraunhofer IGD
+* Timo Sturm, Fraunhofer IGD
+* Rob Taglang, Cesium
+* Maik Thöner, Fraunhofer IGD
+* Steven Vergenz, AltspaceVR
+* Corentin Wallez, Google
+* Alex Wood, Analytical Graphics, Inc
+
+
+[appendix]
+
+[[appendix-a-tangent-space-recalculation]]
+= Tangent Space Recalculation
+
+*TODO*
+
+
+[[appendix-b-brdf-implementation]]
```

Comment:
It looks like each Appendix maybe needs its own `[appendix]` indicator?

Reply:
Yes, thanks. Fixed now.